src/binary.ts

```diff
@@ -313,13 +313,13 @@ export class Binary {
 
   /** @internal */
   static fromExtendedJSON(
-    doc: { $type: string; $binary: string | { subType: string; base64: string } },
```

Comment:
This seemed to be a mistake here, the `{ subType: string; base64: string }` object wasn't  wrapped in `$binary`. 🌯